### PR TITLE
Skip nsp on node 4 and 5

### PR DESCRIPTION
**Pull request by vweevers**
Alternative for #194.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,10 +12,19 @@ env:
...
+  - node_js: '7'
+    env: [NSP=true]
+  - node_js: '6'
+    env: [NSP=true]
+  - node_js: '5'
+  - node_js: '4'
+
+script:
+  - npm run test
+  - test $NSP && npm run audit
```
Comment (vweevers) [inline .travis.yml]: Need a `|| echo "skip nsp"` at the end.

File: .travis.yml [hunk 2]
```diff
@@ -12,10 +12,19 @@ env:
...
+  - node_js: '7'
+    env: [NSP=true]
+  - node_js: '6'
+    env: [NSP=true]
+  - node_js: '5'
+  - node_js: '4'
+
+script:
+  - npm run test
+  - test $NSP && npm run audit || echo "skip audit"
```
Comment (ralphtheninja) [inline .travis.yml]: Nice one!

Comment (vweevers) [reply .travis.yml]: Hm wait. If `npm run audit` fails, does it fail the build?

Comment (ungoldman) [reply .travis.yml]: @vweevers Ideally it should, right? Looks like operator precedence might short circuit this check.

Comment (vweevers) [reply .travis.yml]:
```
$ test $nope && test $nope || echo "oops"
oops
$ echo $?
0
```

To ensure the exit code of `npm run audit` is the exit code of the line, we can do:

```
if [ "$NSP" ]; then npm run audit; else echo "skip audit"; fi
```

Comment (vweevers) [reply .travis.yml]: #198

**Pull request merged**